CIP-xxxx/README.md

```diff
@@ -0,0 +1,136 @@
+For example, a CIP may require that a particular field is always tagged.
+They may also define that a field need not be tagged if it is the typical data structure.
+Only less common or unusual data structures would be tagged in this case.
+A CIP may also define that Tags are not used, and only the canonical encoding.
+
+### Canonical Encoding
+
+When a Tag is defines, its Canonical encoding must also be defined.
+This is to ensure that all data that is tagged is encoded in a uniform manor.
+Even if a CIP does NOT use a tag, it should preferably use the canonical encoding for the data structure.
+This is to prevent fragmentation and confusion amongst compliant encoders and decoders of the various data structures.
+
+If a pre-existing data structure is being tagged, then its most common current encoding should be used as the Canonical encoding.
+It should not be re-defined.
+
+For example `ED25519-BIP32` public keys are commonly encoded as a byte array.
+The array is 32 bytes long with the most significant byte of the key appearing first in the array.
+If a Tag is defined for this public key, it should simply define the Tag.
+Its Canonical encoding MUST follow this common encoding scheme.
+
+If a CIP does not refer to a Tag, nor the Canonical encoding specification for the data structure,
+AND it does not define an alternative encoding.
+Then the application implementing the encoding should assume it is encoded canonically.
+This helps ensure backward compatibility with pre-existing CIPs where Tags are not used.
+
+### Tag Registry
+
+Similar to [CIP-0010] this CIP defines a registry of all known tags.
+The format of the registry is defined by the json schema: [CIP Tag Registry Schema].
+New entries should be added to the [CIP Tag Registry] in a PR for a CIP that first defines a new CBOR Tag.
+They MUST be updated when the Tag is accepted or rejected by IANA.
+The registry clearly notes if the tag is currently known to be registered or not.
+If a Tag is not yet registered then any implementor must be aware that its possible the Tag number could change and is not final.
+
+## Rationale: how does this CIP achieve its goals?
+
+Creating a [registry][CIP Tag Registry] for `CIP Tag` values has the following benefit:
+
+1) It makes it easy for developers to know which `CIP Tags` to use, and if they have been registered or not.
+2) It makes it easy to avoid collisions with other standards that use CIP Tags.
+3) It helps CIP authors to find appropriate CIP tags for their use case, or to define new tags.
+
+The process for defining and registering Tags should help provide clarity about how a CIP tag can be defined.
+It also provides clarity on the responsibility CIP authors have to register the tags they create.
+If a CIP author is not prepared to take on that responsibility they should not create a Tag.
+
+## Path to Active
+
+### Acceptance Criteria
+
+* [ ] [CPS-xxxx] is accepted.
+* [ ] At least 1 CIP that defines tag/s for a cardano data structure.
```

Comment:
```suggestion
* [ ] 3 CIPs are accepted into the Tag registry.
```
Could be a little more ambitious .😆

Reply:
Let's discuss this: I have no problems with being ambitious, but I think this can lead to the CIP sitting "unaccepted" for a lot longer.

Reply:
> Is the "Acceptance Criteria" what gates the Status: from moving from Proposed to Accepted?

It is what moves a CIP from `Proposed` -> `Active` status.
For me, it doesn't really matter too much, I just prefer multiple different tickboxes as they can show progress towards `Active`.

> What about:

Looks good!

Reply:
Updated as suggested